### PR TITLE
PLF-8173 Displaying users in Manage Users app is slow with lot of users

### PR DESCRIPTION
**Pull request by trongtt**
Remove unnecessary manual flush triggers from reading operations

### CODE REVIEW

File: component/identity/src/main/java/org/exoplatform/services/organization/idm/IDMUserListAccess.java [hunk 1]
```diff
@@ -163,29 +158,26 @@ public int getSize() throws Exception {
...
-        getOrganizationService().flush();
-
         int result;
-
         if (size < 0) {
             if (fullResults != null) {
                 result = fullResults.size();
-            } else if (countAll) {
+            } else {
+              if (countAll) {
```
Comment (thomasdelhomenie) [inline component/identity/src/main/java/org/exoplatform/services/organization/idm/IDMUserListAccess.java]: Is this change required ? I don't see the impact actually.

File: component/identity/src/main/java/org/exoplatform/services/organization/idm/IDMUserListAccess.java [hunk 2]
```diff
@@ -163,29 +158,26 @@ public int getSize() throws Exception {
             Tools.logMethodIn(log, LogLevel.TRACE, "getSize", null);
         }
 
-        getOrganizationService().flush();
```
Comment (thomasdelhomenie) [inline component/identity/src/main/java/org/exoplatform/services/organization/idm/IDMUserListAccess.java]: Isn't the flush required in case we need to do a fetch to get the size (so before listQuery(0, 0)) ?

File: component/identity/src/main/java/org/exoplatform/services/organization/idm/IDMUserListAccess.java (hunk 1 above)
Comment (trongtt) [reply component/identity/src/main/java/org/exoplatform/services/organization/idm/IDMUserListAccess.java]: No, this is not required. It just makes the code easier to read.

File: component/identity/src/main/java/org/exoplatform/services/organization/idm/IDMUserListAccess.java (hunk 2 above)
Comment (trongtt) [reply component/identity/src/main/java/org/exoplatform/services/organization/idm/IDMUserListAccess.java]: I just think the flush should be manually required right after an update operation.

File: component/identity/src/main/java/org/exoplatform/services/organization/idm/IDMUserListAccess.java (hunk 1 above)
Comment (thomasdelhomenie) [reply component/identity/src/main/java/org/exoplatform/services/organization/idm/IDMUserListAccess.java]: ok

File: component/identity/src/main/java/org/exoplatform/services/organization/idm/IDMUserListAccess.java (hunk 2 above)
Comment (thomasdelhomenie) [reply component/identity/src/main/java/org/exoplatform/services/organization/idm/IDMUserListAccess.java]: Actually, we should not have to worry about flushing manually IMO, but I don't know why these manual flushes have been added.
